cdc/processor/pipeline/actor_node.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pipeline
+
+import (
+	"context"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/tiflow/pkg/pipeline"
+)
+
+// ActorNode is an async data process node, it fetches and process table event non-blocking
+// if processing is blocked, the data will be cached and wait next run
+type ActorNode struct {
+	eventStash    *pipeline.Message
+	parentNode    AsyncDataHolder
```

Comment:
may be we can unified the name of an `event` / `data` / `message`?

Reply:
good sugestion